frontend/src/app/libs/common-components/models/hint-menu-item.ts

```diff
@@ -0,0 +1,4 @@
+export interface HintMenuItem {
```

Comment:
EventTileMenuItem - we need to use more specific names which include component name for common-components lib, to avoid confusion of consumers